### DISCUSSION TITLE
Decoding Bytestream without keyframes

### DISCUSSION THREAD

**Discussion by EricHenry** [ralfbiedert/openh264-rs#31] [General]
I have an h264 byte stream that doesn’t have key frames. More specifically it doesn’t have a any IDR nal units. 

I am having trouble decoding the nal units into YUV frames. Is it possible to decode the bytestream if there are not any idr nal units?

**Comment by ralfbiedert** [comment]
I think this question is better asked upstream. In my limited understanding of h264, if you don't have IDRs, it's going to be difficult as the remaining delta frames have nothing to work on.
